content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/clientlibs/editor/js/tabs.js

```diff
@@ -0,0 +1,72 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/* global jQuery, Coral */
+(function($, Coral) {
+    "use strict";
+
+    var dialogContentSelector = ".cmp-tabs__editor";
+    var childreneditorSelector = "[data-cmp-is='childrenEditor']";
+    var activeItemSelector = "[data-cmp-tabs-v1-dialog-edit-hook='activeItem']";
+    var activeSelectSelector = "[data-cmp-tabs-v1-dialog-edit-hook='activeSelect']";
+
+    $(document).on("dialog-loaded", function(e) {
+        var $dialog = e.dialog;
+        var $dialogContent = $dialog.find(dialogContentSelector);
+        var dialogContent = $dialogContent.length > 0 ? $dialogContent[0] : undefined;
```

Comment:
I would rename: 
`dialogContent` -> `tabsEditor`
`$dialogContent` -> `$tabsEditor`

Reply:
Done in https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/317/commits/bdd641e41c0331b00636b7113b89f0432fc0c225